File: CrowdInvestWebApp/src/app/models/InvestmentRequestResult.ts
import {UserInvestmentResultType} from "./UserInvestmentResultType";
import {InvestmentFundContribution} from "./InvestmentFundContribution";


export class InvestmentRequestResult {
    requestId: string;
    result: UserInvestmentResultType;
    contribution: InvestmentFundContribution;
}

File: CrowdInvestCore/Models/InvestmentRequestResult.cs
using System;
using CrowdInvestCore.Models.Enums;

namespace CrowdInvestCore.Models
{
	public class InvestmentRequestResult
	{
		public Guid RequestId { get; set; }
		public ResultType Result { get; set; }
		public InvestmentFundContribution Contribution { get; set; }
	}
}